ttnn/cpp/ttnn/operations/data_movement/transpose/device/transpose_op.hpp

```diff
@@ -28,8 +28,11 @@ struct Transpose {
     operation::ProgramWithCallbacks create_program(const std::vector<Tensor>& input_tensors, std::vector<Tensor> &output_tensors) const;
     TransposeOpParallelizationStrategy get_parallelization_strategy(const std::vector<Tensor> &input_tensors) const;
 
-    const operation::Hash compute_program_hash(
-        const std::vector<Tensor> &input_tensors) const;
+    static constexpr auto attribute_names = std::forward_as_tuple("dim", "output_mem_config");
```

Comment:
You shouldn't need `attribute_names` and `attribute_values` at all anymore

Reply:
deleted